src/adapters/IDB.js

```diff
@@ -205,12 +205,30 @@ export default class IDB extends BaseAdapter {
    * @override
    * @return {Promise}
    */
-  list() {
+  list(params={}) {
```

Comment:
we should provide default property values for the params object; eg `Object.keys(params.filters)` can fail when no `filters` object is provided.